src/transformers/models/speech_to_text/modeling_tf_speech_to_text.py

```diff
@@ -187,23 +188,22 @@ def _get_embedding(num_embeddings: int, embedding_dim: int, padding_idx: Optiona
             # zero pad
             emb = tf.concat([emb, tf.zeros(num_embeddings, 1)], axis=1)
         if padding_idx is not None:
-            emb = tf.concat([emb[:padding_idx, :], tf.zeros((1, emb.shape[1])), emb[padding_idx + 1 :, :]], axis=0)
+            emb = tf.concat(
+                [emb[:padding_idx, :], tf.zeros((1, shape_list(emb)[1])), emb[padding_idx + 1 :, :]], axis=0
```

Comment:
Same comment as above for `shape_list` -- `tf.shape(emb)` is a potential replacement

Reply:
Updated :)